test/general/util.js

```diff
@@ -108,19 +108,23 @@ export default () => describe('Util unit tests', function() {
       const data = 'test@example.com';
       expect(util.isEmailAddress(data)).to.be.true;
     });
-    it('should return true for valid email address', function() {
+    it('should return true for valid email address (-- in domain part)', function() {
       const data = 'test@xn--wgv.xn--q9jyb4c';
       expect(util.isEmailAddress(data)).to.be.true;
     });
-    it('should return false for invalid email address', function() {
-      const data = 'Test User <test@example.com>';
-      expect(util.isEmailAddress(data)).to.be.false;
+    it('should return true for valid email address (trailing numbers in domain)', function() {
+      const data = 'test1@com.com09';
+      expect(util.isEmailAddress(data)).to.be.true;
     });
-    it('should return false for invalid email address', function() {
+    it('should return false for invalid email address (no . in domain part)', function() {
```

Comment:
```suggestion
    it('should return true for valid email address (no . in domain part)', function() {
```
(I'm not sure we really need to test this, but if we do, it should match the result :)